tasks/publish-to-mrrc/tests/mocks.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
I asked for shebangs to be added in the actual task which you did. But I see you added `#!/usr/bin/env sh` and so it's different from the mock now. They should use the same so that the test is as realistic as possible - otherwise the test will run with `bash` and in production it will run with `sh`.

Now which one to use: We use `bash` in most places because otherwise shellcheck will complain about some things like using `<<<`. But if everything works for you with `sh`, feel free to use that one.

Reply:
Um... I remember I've set to bash but seems there is something wrong with my memory.
Whatever I've changed to bash in new commit.  Sorry for this.

Reply:
ok, no worries, thanks